CHANGELOG.md

```diff
@@ -7,21 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Change error module to include plonk & poseidon err variants [#131](https://github.com/dusk-network/dusk-blindbid/issues/131)
```

Comment:
Use one approach, either you put all the links at the end or all the links in the same line.
This is comment is valid for the entire changelog.